### PR TITLE
add PR labelling

### PR DESCRIPTION
**Pull request by jamieomaguire**

### CODE REVIEW

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request_target
+
+jobs:
+  categorise:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/labeler@v4
+      with:
+        configuration-path: '.github/project-labeler.yml'
```
Comment (raoufswe) [inline .github/workflows/labeler.yml:13]: I'm just guessing, but is the path correct here? 
the file is located in .github/workflows/project-labeler.yml

File: .github/workflows/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        repo-token: '${{ secrets.GITHUB_TOKEN }}'
```
Comment (raoufswe) [inline .github/workflows/labeler.yml:14]: is the token defined in the repo?

Comment (siggerzz) [reply .github/workflows/labeler.yml:14]: Yep, it's a default token provided by Github to allow actions to take actions on the repo, such as automatic PR's / labelling etc.

File: .github/workflows/labeler.yml (hunk 1 above)
Comment (jamieomaguire) [reply .github/workflows/labeler.yml:13]: ... good spot :D

**Comment by jamieomaguire** [comment]
@siggerzz should the labeler take effect in this PR or after merging?

**Comment by jamieomaguire** [comment]
> > @siggerzz should the labeler take effect in this PR or after merging?
> 
> By default, it'll only run for opened, synchronized, or reopened PRs. For testing purposes, I'd probably get it to run on `labeled`
> 
> Once committed, add a label to this PR, and it should trigger the workflow :)
> 
> ```
> on:
>   pull_request:
>     types:
>       - labelled
> ```

I will try your mystic ways

**Review by raoufswe — APPROVED** [review]

**Review by siggerzz — APPROVED** [review]

**Pull request merged**